lightly/api/utils.py

```diff
@@ -81,6 +82,9 @@ def __iter__(self):
 
     def __next__(self):
         if len(self.entries) == 0:
+            # stop iteration if the last chunk was smaller than the page size
+            if self.last_chunk_size < self.page_size:
```

Comment:
@MalteEbner I also built in this safeguard; there should no need to hit the API until there is a empty list returned. As soon as we get less than the pageSize, it should be good

Reply:
Looks sensible to me.